foundation/foundation-upgrade/src/main/resources/migrations.json

```diff
@@ -64,5 +64,17 @@
         "excludes": ["*-docker/*/target/Dockerfile"]
       }
     ]
+  },
+  {
+    "name": "upgrade-helm-chart-files-names-migration",
+    "implementation": "com.boozallen.aissemble.upgrade.migration.HelmChartsNameMigration",
+    "fileSets": [
+      {
+        "includes": [
+          "**/apps/**/Chart.yaml",
+          "**/apps/**/values*.yaml"
```

Comment:
Q: Should this migration apply solely to yamls in apps/ or should the root chart and value files be candidates as well?

Reply:
Also recommend doing a more specific path:
```
"*-deploy/src/main/resources/apps/*/Chart.yaml",
"*-deploy/src/main/resources/apps/*/values*.yaml"
```

Reply:
Have you verified in a downstream project that `values*.yaml` gets the `values-dev.yaml` and `values-ci.yaml` files?

Reply:
@carter-cundiff yes, the migration updates the values-ci.yaml and values-dev.yaml files